### DISCUSSION TITLE
Select dropdown does not show active text when changing text

### DISCUSSION THREAD

**Discussion by PieterxJan** [livewire/livewire#4826] [7. Bugs (archived)]
Context: I'm trying to make a select that automatically changes the text that shows the influence on the total price. It looks like the text-change triggers something that breaks the active state.

## Steps to reproduce

I have made a snippet on the Laravel Livewire playground tool:
https://laravelplayground.com/#/snippets/806fbebb-1155-45cc-942b-bb75399d26af

**Comment by dev** [comment]
Did you manage to fix this issue? I'm facing the same problem.

**Reply by PieterxJan** [reply]
No, haven't found anything yet. Working around the issue at the moment, but would be nice to get this fixed :)

**Comment by dev** [comment]
Still didn't find a fix for this. @calebporzio any idea?

**Comment by joshhanley** [comment]
@PieterxJan and @dev I've fixed a similar issue in PR #5129 can you see if that fixes this issue too? Thanks!
